lightwood/mixer/neural.py

```diff
@@ -282,8 +284,9 @@ def fit(self, train_data: EncodedDs, dev_data: EncodedDs) -> None:
         criterion = self._select_criterion()
         scaler = GradScaler()
 
-        self.model, epoch_to_best_model, err = self._max_fit(
-            train_dl, dev_dl, criterion, optimizer, scaler, self.stop_after, return_model_after=20000)
+        self.model, epoch_to_best_model, _ = self._max_fit(
+            train_dl, dev_dl, criterion, optimizer, scaler, self.stop_after * 0.8 - (time.time() - self.started),
```

Comment:
@paxcema mentions 10%; i'm assuming this is within 80% of the budget? It's worth keeping a documentation note